odoo/addons/base/views/res_partner_views.xml

```diff
@@ -208,7 +208,7 @@
                             <field name="mobile" widget="phone"/>
                             <field name="user_ids" invisible="1"/>
                             <field name="email" widget="email" context="{'gravatar_image': True}" attrs="{'required': [('user_ids','!=', [])]}"/>
-                            <field name="website" widget="url" placeholder="e.g. https://www.odoo.com"/>
```

Comment:
Wouldn't it be better to replace it with https://www,example.com ? Same above

Reply:
The get set to "https://www.unipart.com", after udes_branding is installed. https://gitlab.unipart.digital/udes/udes_closed_14/-/merge_requests/104/diffs?diff_head=true#:~:text=%3Ee.g.%20https%3A%2F%2Fwww.unipart.com%3Cattribute%2F%3E